pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,78 @@
+# Group Step
+ 
+A *group* step groups up parallel jobs into logical groups and allows displaying them on the build page in the Buildkite UI.  
+
+{:toc}
+ 
+You can scale your build pipelines in Buildkite to support an unlimited number of jobs. However, a large number of jobs on the build page can look messy. If pipelines contain 4-5 steps that can be logically grouped together, you can put them in a group using the `group` step type.  
+ 
+Job groups allow creating and then displaying logical groups in the Buildkite UI. They also help manage dependencies between a collection of steps in a dag (for example, step X depends on everything in a group Y).
+
+<%= image "job-groups-in-UI.png", width: 760, height: 328, alt: "Job groups displayed in the Buildkite UI" %>
+
+ 
+The `group` step type not only groups up parallel jobs and also groups them up by a `group_identifier` if it exists. For example:  
+ 
+``` 
+steps:
+  - command: "echo hello"
+  - group: "hello group"
+    steps:
+      - command: "echo hello"
+      - wait
+      - command: "echo hello"
+        parallelism: 3
+  - group: ~
+    steps:
+      - command: "echo hello"
+      - wait
+      - command: "echo hello"
+  - command: "echo goodbye"
+    parallelism: 2
+```
+ 
+The `group` step type:  
+ 
+* Internally switches the build to `dag` mode
+* Makes groups run in parallel next to each other
```

Comment:
Is this true? I might have misunderstood.

Reply:
Which item on the list? :-D Let's discuss this.